lib/sensors/include/UltraSonicSensor.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Pwmln.h"
+#include "mbed.h"
+
+// declare constants
+const int NUM_SAMPLES{1};
+const int CONV_FACTOR{1000000};
```

Comment:
Where is this conv factor from? what does it represent? please have a better name